hummingbot/strategy/amm_arb/amm_arb_config_map.py

```diff
@@ -88,7 +88,7 @@ def order_amount_prompt() -> str:
         prompt="How much buffer do you want to add to the price to account for slippage for orders on the first market "
                "(Enter 1 for 1%)? >>> ",
         prompt_on_new=True,
-        default=Decimal("0"),
+        default=Decimal("0.05"),
```

Comment:
There are a few reasons for this:
- Hummingbot always round the price down, for buy order this causes the order to be below the quote price and the transaction might get rejected.
- Ethereum transactions are costly, it'd be safer to always provide some margin for price.

Reply:
The quantize function always rounds down and the order price is less in few decimal points in value than the actual quote price. This causes the 'Price too high' on Balancer trade submission. So we set the default to the minimum of 0.05. This also gets applied to Uniswap since the config map is not connector specific.